doc/source/user_guide/ug_case_settings.rst

```diff
@@ -162,6 +162,8 @@ setup_nml
    "", "``m``", "write restart every ``dumpfreq_n`` months", ""
    "", "``y``", "write restart every ``dumpfreq_n`` years", ""
    "", "``1``", "write restart every ``dumpfreq_n`` time step", ""
+   "``dumpfreq_base``", "init", "restart output frequency is relative to the model initial date", "init"
```

Comment:
This is a very good question.  'init' means relative to year_init, month_init, day_init.  So while those generally do not play a role after the initial run, they now do if freq_base is 'init'.  Is there a reason to support a mode where we create a freq_base relative to the start of the current run.  It probably would not be hard to do that.  It could create chaos in the sense that if you write output every 4 days and run 10 days at a time, you'd have output on day 4, 8, 14, 18 in that mode.  In zero mode you'd have it every 4 days with a possible phase shift (i.e. 1, 5, 9, 13, 17).  And in init mode you'd have it on days 4, 8, 12, 16.  As I alluded to, there are so many things we could do.

Let me know if we need another mode here.

Reply:
@eclare108213 and I decided we didn't need another mode at this time.  Documentation has been updated to clarify.